README.md

```diff
@@ -49,7 +49,7 @@ PureScript package manager and build tool powered by [Dhall][dhall] and
 
 ## What does all of this mean?
 
-`spago` aims to tie together the UX of developing a PureScript project.  
+`spago` aims to tie together the UX of developing a PureScript project.
```

Comment:
It looks like "whitespace removal" is a common feature in editors by now, but these two spaces are Markdown syntax (newline without new paragraph) 😅

Could you undo the whitespace changes to the README?

Reply:
◀️